### PR TITLE
Harmonise color on front-end

### PR DESCRIPTION
**Pull request by corentindrouet**
It add the spectrum-color-picker component,
class to css, and template
and a live previous of the color selected.

Add a function on Properties controller to calculated a lighten and darken color when the color is choose,
and the page is refreshed.

### CODE REVIEW

**Comment by dynamiccast** [comment]
Few spots are still no customizable:
- Help tooltip, copy tooltip on user profil
- Edit icon on profile
- Group checkbox to add members/applications
- Stay clicked on a button while leaving hover area
- Input border on focus
- Team create button
- Alert box background (machine page where there is no machines)
- Reset password page
- Default group link in configuration

**Comment by dynamiccast** [comment]
It would be nice to auto refresh the page once "choose" is clicked.

**Comment by dynamiccast** [comment]
Configuration fields need to be bordered in green.

**Comment by dynamiccast** [comment]
One more mistake. Try to stay clicked on the upper right logout button. It's border remains blue.

File: assets/app/styles/global.scss [hunk 1]
```diff
@@ -79,8 +79,84 @@ h1 {
...
+    color: $blue-default;
+    &:hover {
+      color: $blue-lighten;
+    }
+  }
+}
+
+.nav-link .background-nav-default {
+  &.active {
+    background-color: #eef4f9;
```
Comment (spacewoox) [inline assets/app/styles/global.scss]: -> background-color: $gray-lighter;

File: assets/app/styles/global.scss [hunk 2]
```diff
@@ -79,8 +79,84 @@ h1 {
...
+  }
+  &:focus {
+    color: $blue-darken;
+  }
+}
+
+.btn-color-default {
```
Comment (spacewoox) [inline assets/app/styles/global.scss:113]: &:hover,
&:focus,
&:active {
  background-color: $blue-darken;
}

File: assets/app/styles/global.scss [hunk 3]
```diff
@@ -79,8 +79,84 @@ h1 {
...
+  &:hover {
+    background-color: $blue-lighten;
+    border-color: $blue-darken;
+  }
+  &:focus {
+    background-color: $blue-darken;
+  }
+  &:active {
+    background-color: $blue-darken;
+    &:hover {
```
Comment (spacewoox) [inline assets/app/styles/global.scss]: useless

File: assets/app/styles/global.scss [hunk 4]
```diff
@@ -79,8 +79,84 @@ h1 {
...
+      background-color: $blue-lighten;
+      border-color: $blue-default;
+    }
+  }
+  li {
+    &:hover {
+      background-color: $blue-lighten;
+    }
+  }
+  border-color: $blue-lighten;
```
Comment (spacewoox) [inline assets/app/styles/global.scss]: should be on top

File: assets/app/styles/global.scss [hunk 5]
```diff
@@ -79,8 +79,84 @@ h1 {
...
+}
+
+.icon-coloir-default {
```
Comment (spacewoox) [inline assets/app/styles/global.scss]: coloir

File: assets/app/reset-password/token/template.hbs [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 		<div class="section">
 		  <div class="content-wrapper form">
 		    <form {{action 'update' on='submit'}}>
 				<h4>Reset your password</h4>
 		        	{{nano-input type="password" placeholder='New password' model=model valuePath="password"}}
-		        <button type="submit" class="btn btn-primary btn-block text-uppercase">Update my password</button>
+		        <button type="submit" class="btn btn-primary btn-block text-uppercase btn-color-default">Update my password</button>
 				  {{#link-to 'login'}}Sign in{{/link-to}}
-        </form>
+          </form>
```
Comment (spacewoox) [inline assets/app/reset-password/token/template.hbs]: wrong indent

**Review by spacewoox — CHANGES_REQUESTED** [review]

**Pull request merged**